app/src/main/java/org/simple/clinic/user/UserSession.kt

```diff
@@ -118,10 +124,11 @@ class UserSession @Inject constructor(
   }
 
   fun storeUser(user: User, facilityUuid: UUID): Completable {
+    listOf(facilityUuid)
```

Comment:
Similarly here, dangling list? I'm guessing this is a refactoring mistake?

Reply:
Yup, fixing.